packages/larva-tokens/build/sportico.map.scss

```diff
@@ -0,0 +1,61 @@
+
+$sportico-map: (
+  'color-white': (rgb(255, 255, 255)),
+  'color-grey-light': (rgb(220, 220, 220)),
+  'color-grey': (rgb(140, 140, 140)),
+  'color-grey-dark': (rgb(140, 140, 140)),
+  'color-black': (rgb(0, 0, 0)),
+  'color-brand-primary': (rgb(178, 42, 45)),
+  'color-brand-secondary': (rgb(171, 202, 89)),
+  /* The webfont for font-family-primary. */
+  'font-family-primary-fancy': (Geograph, serif),
+  /* The fallback font for font-family-primary. */
+  'font-family-primary': (Georgia, serif),
+  /* Available via Typekit. */
+  'font-family-secondary-fancy': (Druk Wide, serif),
+  /* Fallback from secondary-fancy */
+  'font-family-secondary': (Georgia, serif),
+  'font-family-body': (Georgia, serif),
+  'font-family-basic': (Helvetica, sans-serif),
+  'background-color-grey-light': (rgb(204, 204, 204)),
```

Comment:
Not that this is different that the `color-grey-light` - we need to have background color assigned specifically also.